tests/functional/modules/test_zos_fetch_func.py

```diff
@@ -664,3 +666,133 @@ def test_fetch_flat_create_dirs(ansible_zos_module, z_python_interpreter):
     finally:
         if os.path.exists(dest_path):
             shutil.rmtree("/tmp/" + remote_host)
+
+
+def test_fetch_sequential_data_set_with_special_chars(ansible_zos_module):
+    hosts = ansible_zos_module
+    TEST_PS = "USER.TEST.@N$IBLE#"
```

Comment:
I recommend for the name with symbols we have the option of get_tmp_ds_name(symbols=True)

Reply:
Done